src/Catalog/CatalogIndexEntry.cs

```diff
@@ -84,6 +96,11 @@ public int CompareTo(CatalogIndexEntry other)
             return CommitTSComparer.Compare(this, other);
         }
 
+        public bool IsDelete()
+        {
+            return Types.Any(type => type == "nuget:PackageDelete");
```

Comment:
didn't notice the class was designed like that...why do we have `Type` return an `IEnumerable`?